packages/flutterfire_ui/lib/src/i10n/lang/nl.dart

```diff
@@ -0,0 +1,263 @@
+    'Het wachtwoord is fout of de gebruiker heeft geen wachtwoord',
+    this.signInText = 'Login',
+    this.registerText = 'Registreer',
+    this.registerHintText = "Nog geen account?",
+    this.signInHintText = 'Al een account?',
+    this.signOutButtonText = 'Uitloggen',
+    this.phoneInputLabel = 'Telefoonnummer',
+    this.phoneNumberInvalidErrorText = 'Telefoonnummer is fout',
+    this.phoneNumberIsRequiredErrorText = 'Telefoonnummer is verplicht',
+    this.profile = 'Profiel',
+    this.name = 'Naam',
+    this.deleteAccount = 'Verwijder account',
+    this.passwordIsRequiredErrorText = 'Wachtwoord is verplicht',
+    this.confirmPasswordIsRequiredErrorText = 'Bevestig wachtwoord',
+    this.confirmPasswordDoesNotMatchErrorText = 'Wachtwoorden komen niet overeen',
+    this.confirmPasswordInputLabel = 'Bevestig wachtwoord',
+    this.forgotPasswordButtonLabel = 'Wachtwoord vergeten?',
+    this.forgotPasswordViewTitle = 'Wachtwoord vergeten',
+    this.resetPasswordButtonLabel = 'Wachtwoord herstellen',
+    this.verifyItsYouText = "Verifier dat jij het bent",
+    this.differentMethodsSignInTitleText =
+    'Gebruik één van de volgende methodes om in te loggen',
+    this.findProviderForEmailTitleText = 'Voer uw e-mailadres in om verder te gaan',
+    this.continueText = 'Verder',
+    this.countryCode = 'Landcode',
+    this.codeRequiredErrorText = 'Landcode is verplicht',
+    this.invalidCountryCode = 'Ongeldige landcode',
+    this.chooseACountry = 'Kies een land',
+    this.enableMoreSignInMethods = 'Meer inlogmethodes aanzetten',
+    this.signInMethods = 'Inlogmethodes',
+    this.provideEmail = 'Vul e-mail en wachtwoord in',
+    this.goBackButtonLabel = 'Terug',
+    this.passwordResetEmailSentText =
+    "We hebben je een e-mail met een link gestuurd om je wachtwoord te herstellen. Controleer je e-mail.",
+    this.forgotPasswordHintText =
+    'Vul je e-mail in en we sturen je een mail om je wachtwoord te herstellen',
+    this.emailLinkSignInButtonLabel = 'Login met een magic link',
+    this.signInWithEmailLinkViewTitleText = 'Login met een magic link',
+    this.signInWithEmailLinkSentText =
+    "We hebben een link gestuurd met een magic link. Controleer je e-mail en klik op de link om in te loggen",
+    this.sendLinkButtonLabel = 'Stuur magic link',
+    this.arrayLabel = 'array',
+    this.booleanLabel = 'boolean',
+    this.mapLabel = 'map',
+    this.nullLabel = 'null',
+    this.numberLabel = 'number',
+    this.stringLabel = 'string',
+    this.typeLabel = 'type',
+    this.valueLabel = 'value',
+    this.cancelLabel = 'cancel',
+    this.updateLabel = 'update',
+    this.northInitialLabel = 'N',
+    this.southInitialLabel = 'Z',
+    this.westInitialLabel = 'W',
+    this.eastInitialLabel = 'O',
+    this.timestampLabel = 'timestamp',
+    this.longitudeLabel = 'longitude',
+    this.latitudeLabel = 'latitude',
+    this.geopointLabel = 'geopoint',
+    this.referenceLabel = 'reference',
```

Comment:
Are data types and values generally left untranslated? 👀 
If so, no problem. Just checking 😄

Reply:
These have been translated in other localization PRs so I think they should also be translated here as well if possible please

Reply:
@jeroen-meijer, I just find it weird to translate. Most of them don't really have a translation or one 1 would never use. Then it's weird that some do have a translation which you might not expect.
For example
```
    this.arrayLabel = 'reeks', //would not use it, would be confusing I think
    this.booleanLabel = 'boolean', // Maybe booleaan or booleaans.
    this.mapLabel = 'map', // kind of dutch already
    this.nullLabel = 'null',
    this.numberLabel = 'getal', // It's ok
    this.stringLabel = 'tekst', // not sure if it is really the same 
    this.typeLabel = 'type', // kind of dutch already
    this.valueLabel = 'waarde',
    this.cancelLabel = 'annuleren', //This one I really would translate
    this.updateLabel = 'update', // kind of dutch already
    this.northInitialLabel = 'N',
    this.southInitialLabel = 'Z',
    this.westInitialLabel = 'W',
    this.eastInitialLabel = 'O',
    this.timestampLabel = 'tijdstempel', //would not use it
    this.longitudeLabel = 'lengtegraad', // These are good translations
    this.latitudeLabel = 'breedtegraad',  // These are good translations
    this.geopointLabel = 'geopunt', // never used 'geopunt', might be 'coordinaat', but that's 'coordinate' and slightly different.
    this.referenceLabel = 'referentie', // this could be fine
```

Reply:
I completely agree, just wanted to check if this is the standard across translations 😄 

I would find it quite confusing and annoying to have to use Dutch versions of the data types. In fact, it would honestly most likely prompt me to switch back to English if this were the case 😋 
So yes, I'm all for keeping the English vocab untouched for data types and other such programming constructs 👍🏻